pages/index.html

```diff
@@ -10,7 +10,7 @@
     <li>
       <a href="https://twitter.com/share" class="twitter-share-button"
          data-url="http://www.mixxx.org"
-         data-text="{% blocktrans %}Check out Mixxx - Free MP3 DJ mixing software for Windows, Mac OS X, and Linux{% endblocktrans %}"
+         data-text="{% blocktrans %}Check out Mixxx - Free Digital DJ MP3 mixing software for Windows, Mac OS X, and Linux{% endblocktrans %}"
```

Comment:
"Free" might be an indicator of poor quality. 

How can we make sure that it is "free" in sense of freedom? 
"Free and open source"

Does "digital" add any value? How about remove MP3?